mandatory/4-tax.js

```diff
@@ -17,7 +19,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(priceNoTax) {
+  return "£" + (priceNoTax + (priceNoTax * 0.2)).toFixed(2);
```

Comment:
Suggestion: Rather than re-implementing the sales tax logic in this function it would make sense to call the `calculateSalesTax` function you wrote above